user_software/ClientGUI/data.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<calculation method="sequential">
+  <general>
+    <time_length>15</time_length>
+  </general>
+  <tube>
+    <radius>7</radius>
+    <thickness>2</thickness>
+    <height>45</height>
+  </tube>
+  <material>
+    <tempreture>1200</tempreture>
+    <conductivity>1.2</conductivity>
+  </material>
+  <reservoir>
+    <init_tempreture>100</init_tempreture>
+  </reservoir>
```

Comment:
To make my parser simpler, you mind to get rid of <general>, <tube>, <material>, and <reservoir> tags and make all the properties under <calculation> tag?